oauth2_http/java/com/google/auth/oauth2/OAuth2Utils.java

```diff
@@ -202,5 +214,24 @@ static Map<String, Object> validateMap(Map<String, Object> map, String key, Stri
     return (Map) value;
   }
 
+  /** Helper to convert from a PKCS#8 String to an RSA private key */
+  static PrivateKey privateKeyFromPkcs8(String privateKeyPkcs8) throws IOException {
```

Comment:
you can remove the copy from serviceaccountcredentials now and reference this one

Reply:
Done. Also change the files which use `privateKeyFromPkcs8` in serviceaccountcredentials